welcome/oce_about.adoc

```diff
@@ -0,0 +1,211 @@
+[[about_oce]]
+= About OCE
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+
+.Table 2
+|===
+2+| |Engine |Platform
+
+2+h|Core Kubernetes and container orchestration
+h| X
+h| X
+
+.3+|
+| Enterprise-class Kubernetes
+| X
+| X
+
+| Red Hat Enterprise Linux for Virtual Datacenters and Red Hat CoreOS
+| X
+| X
+
+| Windows containers compatible
+| X
+| X
+
+2+h|Enterprise-ready configurations
+h| X
+h| X
+
+.3+|
+| Complete life-cycle automation of platform
+| X
+| X
+
+
+| Management, monitoring, alerting, and remedy through Prometheus
+| X
+| X
+
+
+| Enterprise-ready security
+| X
+| X
+
+2+h|Standard infrastructure services
+h| X
+h| X
+
+.3+|
+| Edge routing
+| X
+| X
+
+
+| Flat networking
+| X
+| X
+
+
+| Kubernetes storage plug-ins
+| X
+| X
+
+2+h|Core user experience
+h| X
+h| X
+
+.6+|
+| Application deployments, deployment configuration, templates, and Operators
+| X
+| X
+
+| Administrator console
+| X
+| X
+
+| Operator life cycle management
+| X
+| X
+
+| kubectl command line
+| X
+| X
+
+| Developer projects
+| X
+| X
+
+| Prometheus metrics
+| X
+| X
+
+2+h|Maintained and curated content
+h| X
+h| X
+
+.3+|
+| Red Hat Container Catalog access
+| X
+| X
+
+| Independent Software Vendor and partner Operator access
+| X
+| X
+
+| Service Catalog and Brokers for cloud content
+| X
+| X
+
+2+h|Red Hat add-ons
+h| 
+h| X
+
+.7+|
+| OpenShift Container Storage access
+| X
+| X
+
+| Red Hat Middleware access
+| X
+| X
+
+| Serverless cloud functions
+| X
+| X
+
+| Istio runtime launchers
+| 
+| X
+
+| Quay integration
+| X
+| X
+
+| Container Native Virtualization for virtual machines
+| X
+| X
+
+| Multi-cluster federation
+| X
+| X
+
+2+h|Advanced cluster management
+h| 
+h| X
+
+.3+|
+| Logging
+| 
+| X
+
+| Chargeback
+| 
+| X
+
+| Kiali (Jaeger and OpenTracing)
+| 
+| X
+
+2+h|Advanced networking
+h| 
+h| X
+
+.4+|
+| Egress per pod or namespace
+| 
+| X
+
+| Ingress for non-standard ports
+| 
+| X
+
+| Multi-tenant SDN 
+| 
+| X
+
+| Red Hat Service Mesh (Istio)
+| 
+| X
+
+2+h|Developer experience
+h| 
+h| X
+
+.4+|
+| Integrated CI/CD pipelines
+| 
+| X
+
+| Source to image container factory
```

Comment:
Is "Source to image container factory" a product I don't know about? Please double-check the capitalization.

Reply:
This should be Source-to-Image Container factory. Not sure if factory needs to be capitalized.